kordac/tests/SmokeTests.py

```diff
@@ -65,6 +68,9 @@ def test_compile_files(self):
                 self.assertTrue(len(result.html_string) > 0)
 
     def test_compile_files_custom(self):
```

Comment:
git blame tells me that `algorithms.md` and `introduction.md` were last modified last month, are they contain enough of the new processors with different optional attibutes to make this test thorough enough?